buildtest/buildsystem/base.py

```diff
@@ -539,6 +540,7 @@ def _write_test(self, lines=None):
         self.logger.debug(
             f"Applying permission 755 to {self.metadata['testpath']} so that test can be executed"
         )
+        return self.metadata['testpath']
```

Comment:
What is this return added for?

Reply:
this was me undoing my change it's already part of ``devel`` https://github.com/buildntest/buildtest/blob/devel/buildtest/buildsystem/base.py#L554